### PR TITLE
Remove UpdateStatus command from Tenant

### PR DESCRIPTION
**Pull request by miguelTaningcoYoppworks**
### Issue:

- Currently in the Tenant entity, there is an UpdateStatus command. We want to remove this and in its place have commands that explicitly change the state of the entity.

### Changes:

- Remove UpdateStatus
- Remove StatusUpdated
- Add ActivateTenant
- Add SuspendTenant
- Add TenantActivated
- Add TenantSuspended

### Testing:

- validated riddl

### Notes:

- I realized I named the branch wrong, it is supposed to be remove-update-status-from-tenant

### CODE REVIEW

File: src/main/riddl/Tenant/tenantMessages.riddl [hunk 1]
```diff
@@ -19,19 +19,23 @@ described by {
   |about the tenant.
 }
 
-command UpdateStatus is {
+command ActivateTenant is {
 	tenantId: TenantId,
 	updatingUser: String
```
Comment (reid-spencer) [inline src/main/riddl/Tenant/tenantMessages.riddl:24]: updatingUser should be of type Id(Member) and be named activatingMember

File: src/main/riddl/Tenant/tenantMessages.riddl [hunk 2]
```diff
@@ -19,19 +19,23 @@ described by {
...
+event TenantActivated is {
   tenantId: TenantId,
-  newStatus: ^TenantContext.Status,
   meta: TenantContext.MetaInfo
-} briefly "The event resulting from updating a Tenant's status"
+}
+
+command SuspendTenant is {
+	tenantId: TenantId,
+	updatingUser: String
```
Comment (reid-spencer) [inline src/main/riddl/Tenant/tenantMessages.riddl:33]: updatingUser → suspendingMember
String → Id(Member)

File: src/main/riddl/Tenant/tenantTypeDefinitions.riddl [hunk 3]
```diff
@@ -17,21 +17,10 @@ type MetaInfo is {
...
 described as {
   |data kept about an organization that the user does not directly modify.
   |Instead the system updates this info as a Member Owner updates details
   |in the Info type."
-}
-
-type Status is any of {
-  Draft briefly "Tenant is inactive, in process of being created/defined",
-  Active briefly "Tenant is active and suitable for use",
-  Suspended briefly "Tenant is created but administratively suspended"
```
Comment (reid-spencer) [inline src/main/riddl/Tenant/tenantTypeDefinitions.riddl:36]: I think you still need this and its usage on line 21-24 above.  Think of having a query named "GetState" or "GetStatus" ?  The commands to separate the state changes have been added but someone might still need to know what that state is, after it has changed.

**Review by reid-spencer — CHANGES_REQUESTED** [review]
Just a few minor adjustments needed.

**Comment by miguelTaningcoYoppworks** [comment]
Treating this branch as now unnecessary because of IA-128 which completely changes and overwrites this. Will close this branch and PR.

**Pull request closed without merging**